src/main/java/com/jalasoft/sdfc/pages/Home.java

```diff
@@ -0,0 +1,16 @@
+package com.jalasoft.sdfc.pages;
```

Comment:
I do not think that this file is part of the scope of this task, and besides, the "web element" and the method are the same as those found in the Header file.

Reply:
deleted, please review last commit.